automaton/src/test/java/org/killbill/automaton/dot/TestDOTBuilder.java

```diff
@@ -20,95 +20,96 @@
 
 package org.killbill.automaton.dot;
 
+import java.util.Map;
+import java.util.TreeMap;
+
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-import com.google.common.collect.ImmutableMap;
-
 public class TestDOTBuilder {
 
-    @Test(groups = "fast", enabled = false, description = "https://github.com/killbill/killbill-commons/issues/4")
+    @Test(groups = "fast", enabled = true, description = "https://github.com/killbill/killbill-commons/issues/4")
```

Comment:
Looks like this fixes https://github.com/killbill/killbill-commons/issues/4 ? If so, you can simplify the annotation to `@Test(groups = "fast")`